FreeRTOS/FreeRTOS-Plus-TCP
# Add Unit Tests for coverage for FreeRTOS_DNS_Parser

UT for DNS_Parser

Description
-----------
UT added in DNS_Parser for 100% coverage

Test Steps
-----------
```
cmake -S test/unit-test -B test/unit-test/build/ -DCMAKE_BUILD_TYPE=Debug
make -C test/unit-test/build/ all
```
Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
